src/build.py

```diff
@@ -840,6 +858,27 @@ def V8():
     CopyBinaryToArchive(os.path.join(V8_OUT_DIR, a))
 
 
+def Jsc():
+  buildbot.Step('JSC')
+  Mkdir(JSC_OUT_DIR)
+
+  proc.check_call(['xcrun', PREBUILT_CMAKE_BIN, '-Wno-dev',
+                   '..', '-G', 'Ninja',
+                   '-DCMAKE_BUILD_TYPE="Release"',
+                   '-DPORT=Mac',
+                   '-DENABLE_WEBASSEMBLY=ON'],
+                  cwd=JSC_OUT_DIR)
+  proc.check_call(['ninja', 'jsc'], cwd=JSC_OUT_DIR)
+  proc.check_call(['../Tools/Scripts/run-javascriptcore-tests',
```

Comment:
The build was 1117 files on my laptop. Surely goma would be faster than compiling that with -j4?